### DISCUSSION TITLE
Copilot fails to generate anything in Neovim

### DISCUSSION THREAD

**Discussion by speelbarrow** [community/community#70759] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I noticed last night that Copilot had been strangely silent. I tried to get it to generate some code using `:Copilot panel` but it just hangs on '`Synthesizing 0/10 solutions`'. I'm not working behind any kind of proxy or VPN, and the behaviour is consistent across WiFi networks. I installed a JetBrains IDE to test it there, and Copilot was working fine. I also tried rolling back the version of the plugin to no avail. What can I do to further debug this issue?

**Comment by emiliodallatorre** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> I noticed last night that Copilot had been strangely silent. I tried to get it to generate some code using `:Copilot panel` but it just hangs on '`Synthesizing 0/10 solutions`'. I'm not working behind any kind of proxy or VPN, and the behaviour is consistent across WiFi networks. I installed a JetBrains IDE to test it there, and Copilot was working fine. I also tried rolling back the version of the plugin to no avail. What can I do to further debug this issue?

I'm facing the same issue, and I think it's related to my Node.js version (21.2.0). Could you please share yours? And maybe try with Node 16?
I cannot perform this test as I'm on a corporate device with limited admin rights.
